web/src/components/ChecklistTag.tsx

```diff
@@ -0,0 +1,39 @@
+import { ChecklistStatusText, CheckoffStatus } from "@businessnjgovnavigator/shared/license";
+import { ReactElement } from "react";
+import { Icon } from "./njwds/Icon";
+
+interface Props {
+  status: CheckoffStatus;
+}
+
+interface LicenseStatusMetadata {
+  iconName: string;
+  iconColor: string;
+  statusText: ChecklistStatusText;
+}
+
+const statusObj = new Map<CheckoffStatus, LicenseStatusMetadata>([
+  ["ACTIVE", { iconName: "check", iconColor: "text-success", statusText: "Completed" }],
+  ["PENDING", { iconName: "schedule", iconColor: "base", statusText: "Pending" }],
+  ["INCOMPLETE", { iconName: "close", iconColor: "base", statusText: "Incomplete" }],
+  ["SCHEDULED", { iconName: "event", iconColor: "base", statusText: "Scheduled" }],
+  ["NOT_APPLICABLE", { iconName: "do_not_disturb", iconColor: "base", statusText: "N/A" }],
+  ["UNKNOWN", { iconName: "do_not_disturb", iconColor: "base", statusText: "N/A" }],
+]);
+
+export const ChecklistTag = (props: Props): ReactElement => {
+  const { iconName, iconColor, statusText } = statusObj.get(props.status) ?? {
+    iconName: "",
+    iconColor: "",
+    statusText: "N/A",
+  };
+
+  return (
+    <div className="flex flex-align-center font-sans-2xs width-full tablet:width-auto line-height-sans-2 bg-base-lighter padding-y-05 padding-x-1 padding-right-2 radius-md">
```

Comment:
[pebble] thoughts on `padding-left-1 padding-right-3` instead of `padding-x-1 padding-right-2` because it might be a little confusing as the `padding-x-1` also contributes to the `padding-right`?